canonical/ubuntu.com
# Remove unwanted openstack versions from chart-data.js

## Done

- Remove 'Rock, Stein, Train, Ussuri, Victoria, Xena' from the chart based on [Tytus' comments](https://warthogs.atlassian.net/browse/WD-3155) 

## QA

- Go to https://ubuntu-com-12848.demos.haus/about/release-cycle#openstack-release-cycle and check that the openstack releases mentioned above have been removed (not the LTS versions)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3155
